src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsuserpreferences/controller/CourtsController.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.hmpps.hmppsuserpreferences.controller
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.hmppsuserpreferences.jpa.entity.Users
+import uk.gov.justice.digital.hmpps.hmppsuserpreferences.jpa.repository.UsersRepository
+import java.util.UUID
+
+@RestController
+class CourtsController {
+
+  @Autowired
+  lateinit var repository: UsersRepository
+
+  @ResponseStatus(value = HttpStatus.CREATED)
```

Comment:
Does the DPS project template bring in the Swagger components ? We could do with swagger annotations.